addons/hr_payroll/views/hr_benefit_views.xml

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record model="ir.actions.server" id="generate_benefit">
+        <field name="name">Generate Benefit</field>
+        <field name="model_id" ref="hr.model_hr_employee"/>
+        <field name="binding_model_id" ref="hr.model_hr_employee"/>
+        <field name="state">code</field>
+        <field name="code">
+if records:
+    records.generate_benefit()
+        </field>
+    </record>
+
+    <record id="hr_benefit_action" model="ir.actions.act_window">
+        <field name="name">Benefit</field>
+        <field name="res_model">hr.benefit</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">calendar,tree,form</field>
+    </record>
+
+    <record id="hr_benefit_calendar" model="ir.ui.view">
+        <field name="name">hr.benefit.calendar</field>
+        <field name="model">hr.benefit</field>
+        <field name="arch" type="xml">
+            <calendar string="Benefit" date_start="date_start" date_stop="date_stop" mode="month" quick_add="False" color="benefit_type_id">
+                <field name="state" invisible="1"/>
+                <field name="employee_id" invisible="1"/>
+                <field name="error" invisible="1"/>
+                <field name="name"/>
+                <templates>
+                    <t t-name="calendar-box">
+                        <t t-set="color" t-value="widget.getColor(event.color_index)"/>
+                        <div t-att-style="typeof color === 'string' ? ('background-color:'+color)+';' : ''" t-attf-class="#{record.is_highlighted &amp;&amp; record.is_highlighted.value ? 'o_event_hightlight' : ''} #{typeof color === 'number' ? 'o_calendar_color_'+color : ''}">
+                            <field name="name"/>
+                            <div class="fc-time"/>
+                            <div class="o_fields">
+                                <t t-if="record.error">
+                                    <span class="fa fa-exclamation-triangle" aria-label="Conflict" title="error"/>
+                                    <t t-esc="' '"/> <!-- any better way to add a space? ´&nbsp;´ is not recognized :( -->
```

Comment:
I think you have to use a class to add a margin on the span something like `mr-2`
which is a standard bootstrap 4 class